frontend/src/component/feature/FeatureStrategy/FeatureStrategyConstraints/EditableConstraint.tsx

```diff
@@ -386,22 +400,19 @@ export const EditableConstraint: FC<Props> = ({
                     </StyledIconButton>
                 </HtmlTooltip>
             </TopRow>
-            {showInputField ? (
-                <InputContainer>
-                    <ResolveInput
-                        setValues={setValues}
+            {inputType === 'legal values' ? (
+                <LegalValuesContainer>
+                    <LegalValuesSelector
```

Comment:
We don't need resolveinput anymore, because the legal value input is the only input type we want.